en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,184 @@
+Medias display
+==============
+
+Context
+-------
+
+To improve your website performances, a common way is to download the page data from
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url. Recent browser (such as ``Chrome`` or ``Firefox`` can perform
+up to 10 simultaneous requests). In the following documentation, we would just assume that
+browsers can make only two simultaneous request to the same domain.
```

Comment:
we will just assume
requests
Your assumption is not realistic, as you said above. I don't see the point of doing it.